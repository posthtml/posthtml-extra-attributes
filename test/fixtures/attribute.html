<div role="section">test</div>
