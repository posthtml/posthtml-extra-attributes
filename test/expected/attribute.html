<div role="section" aria-roledescription="slide">test</div>
